content/docs/observable.md

```diff
@@ -0,0 +1,202 @@
+# Observable
+
+> [tip box yellow] This functionality was added in v2.0.0-alpha.202 and is still experimental.
+
+### TLDR
+Imba provides MobX-like reactivity implemented as decorators.
+- Mark class and tag members as observable with the `@observable` decorator.
+- Methods decorated with `@autorun` will be reactively invoked whenever
+referenced observables change.
+- Getters decorated with `@computed` will be cached on first use
+and only recomputed whenever referenced observables change.
+
+See also [additional details](#additional-details).
+
+### Guide
+In Imba, it's important to keep in mind that [commit](/api/imba/commit) is called after every handled event.
+This means that state changes which occur inside of event handlers will be reflected in the next render.
+We can apply this knowledge to implement a simplistic search:
+#### The Inefficient Approach
```

Comment:
These headers need a blank line below them so they don't run up against the code example below them.

Reply:
I'm realizing that's just how it works, need to address through CSS separately.